apps/backend/src/modules/quiz-category/quiz-category.repository.ts

```diff
@@ -0,0 +1,44 @@
+import { type Repository } from "~/libs/types/types.js";
+
+import { QuizCategoryEntity } from "./quiz-category.entity.js";
+import { type QuizCategoryModel } from "./quiz-category.model.js";
+
+class QuizCategoryRepository implements Repository {
+	private quizCategoryModel: typeof QuizCategoryModel;
+
+	public constructor(quizCategoryModel: typeof QuizCategoryModel) {
+		this.quizCategoryModel = quizCategoryModel;
+	}
+
+	create(): Promise<unknown> {
+		throw new Error("Method not implemented.");
+	}
+
+	delete(): Promise<boolean> {
+		throw new Error("Method not implemented.");
+	}
+
+	async find(id: number): Promise<null | QuizCategoryEntity> {
+		const quizCategory = await this.quizCategoryModel.query().findById(id);
+
+		if (!quizCategory) {
```

Comment:
DB will return null if dont find anything
You don't need check if it's null or not

Reply:
DB returns `undefined` if there's nothing